package.json

```
@@ -11,6 +11,8 @@
     "docs:serve": "cd ./docs && yarn serve",
     "docs:deploy": "cd ./docs && yarn deploy",
     "lint": "./scripts/lint",
+    "format:scss": "find ./src -type f -name '*.scss' | xargs yarn prettier --write ",
```

```suggestion
    "format:scss": "prettier --write ./src/**/*.scss",
```
The `./**/*.scss` should grab all `.scss` files in directories and subdirectories.